datasets/gov/vmi/ja_nepriemokos.csv

```diff
@@ -0,0 +1,21 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/vmi/ja_nepriemokos,,,,,,,,,,,,,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,vadovas,,,,,https://atviriduomenys.readthedocs.io/,,
+,,,,,,,,,,,,,,
+,,,,NepriemokosSuma,,,,,,,,,Juridinių asmenų nepriemokos sumos,
+,,,,,id,string,,,,4,open,,Unikalus įrašo ID,
+,,,,,ins_data,date,D,,,4,open,,Įrašo sukūrimo data ir laikas,
+,,,,,upd_data,date,D,,,4,open,,Įrašo atnaujinimo data ir laikas,
+,,,,,ja_kodas,string,,,,3,open,,Juridinio asmens (toliau – JA) kodas,
+,,,,,ja_pavadinimas,string,,,,3,open,,JA pavadinimas,
```

Comment:
1. Kodėl nurodote 3 brandos lygį?
2. Ar tai suprasti, kaip nuasmenintus duomenis?
![image](https://github.com/atviriduomenys/manifest/assets/85924252/cb87a675-94b2-4547-b52c-f92def8c98ca)

3. Mano nuomone, tai yra 2 brandos lygio duomenys, net nurodžius siejimas su JAR, nes siejimas per pavadinimą yra nepatikimas.
Nepaisant to, kad brandos lygis bus 2, rekomenduoju nurodyti galima siejimą patikslinant `type` ir `ref`.

https://atviriduomenys.readthedocs.io/dsa/level.html
https://atviriduomenys.readthedocs.io/dsa/ref.html#brandos-lygis